variables.tf

```diff
@@ -60,6 +60,22 @@ variable "db_port" {
   description = "Database port"
 }
 
+variable "manage_admin_user_password" {
```

Comment:
Is there any reason to keep `admin` in the variables names instead of `master`?
I'd rename `manage_admin_user_password` -> `manage_master_user_password` for the consistency.

Reply:
It's to keep it consistent with the rest of the inputs for the module (see [user](https://github.com/cloudposse/terraform-aws-rds-cluster?tab=readme-ov-file#input_admin_user) and [password](https://github.com/cloudposse/terraform-aws-rds-cluster?tab=readme-ov-file#input_admin_password)).

Reply:
Got it, makes sense.